lib/export/coldp/files/type_material.rb

```diff
@@ -0,0 +1,153 @@
+# ID
+# nameID
+# citation
+# status
+# referenceID
+# locality
+# country
+# latitude
+# longitude
+# altitude
+# host
+# date
+# collector
+# institutionCode
+# catalogNumber
+# associatedSequences
+# sex
+# link
+# remarks
+#
+module Export::Coldp::Files::TypeMaterial
+
+  def self.locality(co)
+    [co.collecting_event&.cached_level0_geographic_name, co.collecting_event&.cached_level1_geographic_name, co.collecting_event&.cached_level2_geographic_name].compact.join(', ')
+  end
+
+  # TODO: This very likely includes more than just hosts (symbiotypes):
+  #   could possibly be narrowed down by checking if the biological relationship type's inverted name includes the word host?
+  #     we are specifying the relationship type in the string to make it clearer
+  def self.host(co)
+    if co.biological_associations == []
+      return nil
+    end
+    hosts = []
+    co.biological_associations.each do |ba|
```

Comment:
You should add the 'Otu' constraint here if that's what you want, like:

`co.biological_association.where(biological_association_subject_type: 'Otu')`